tests/integration/routes/test_feedback.py

```diff
@@ -309,6 +309,32 @@ def test_feedback_submitted_done_button_after_email_confirmation(self):
         selector = "[data-qa='btn-done']"
         self.assertInSelector("/submitted/thank-you", selector)
 
+    def test_feedback_dropdown_form_type(self):
```

Comment:
I am unsure what this is testing and it looks like there are 2 tests in one here. For me this is exercising the details answer using questionnaire form, but do we not test that already? We naturally need one feedback form with a topic, but I don't think we need multiple

Reply:
Added two new testes as you recommended, test schemas removed, irrelevant tests removed! Thanks!